common.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# Copyright (c) 2012-2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+IMAGE_AGENT=workspace-data-sync-agent
+IMAGE_STORAGE=workspace-data-sync-storage
+TAG=$1
+ORGANIZATION=$2
+
+function compile() {
+  echo "Compile file sync progress watcher binary from source code"
+  $(GOOS=linux go build -o ./dockerfiles/agent/scripts/watcher ./watcher/watcher.go)
```

Comment:
Why not compile binary in docker with the multistage build?

Reply:
create issue for this:
https://github.com/che-incubator/workspace-data-sync/issues/4